.vscode/settings.json

```diff
@@ -23,5 +23,6 @@
   },
   "javascript.preferences.importModuleSpecifier": "relative",
   "typescript.preferences.importModuleSpecifier": "relative",
+  "editor.formatOnSave": true,
```

Comment:
I agree with the auto lint options. But this is not related code with `Badge` feature. Could you split this to another PR?

Reply:
@JeffGuKang Can you elaborate what you meant by 'not related code with `Badge`' ? our team members seem to be not really familiar with settings.json codes.

Reply:
`vscode/settings.json` is not related with Badge feature. Thus, do not import the file in this PR.